src/test/java/com/google/jenkins/plugins/computeengine/integration/ITUtil.java

```diff
@@ -141,7 +142,10 @@ private static String loadCredentialsString() {
   }
 
   static Credentials initCredentials(JenkinsRule r) throws Exception {
-    ServiceAccountConfig sac = new StringJsonServiceAccountConfig(CREDENTIALS);
+    SecretBytes bytes = SecretBytes.fromBytes(CREDENTIALS.getBytes(StandardCharsets.UTF_8));
```

Comment:
This logic seems to be repeated in a few places throughout the code. Does it make sense to consolidate?

Reply:
Done